core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -316,6 +322,11 @@ public Map<StoredTabletFile,DataFileValue> getFilesMap() {
     return files;
   }
 
+  public long getFileSize() {
```

Comment:
```suggestion
   /**
    * @return the sum of the tablets files sizes
    */
  public long getFileSize() {
```


Would be nice to add a unit test for this new method.

Reply:
Yeah, I still need to do some more testing updates for the ITs as well so i'll add it at the same time